src/utils/rest-api.js

```diff
@@ -407,6 +407,33 @@ export function fetchRootFolders(types) {
     return backendFetchJson(fetchRootFoldersUrl);
 }
 
+export function fetchElementsMetadata(ids, elementTypes, equipmentTypes) {
```

Comment:
Why not using existing fetchElementsInfos function ? 
Here, there is just an additional parameter equipmentTypes to deal with, but the endpoint called in explore-server is the same ...

Reply:
yes. good catch